### ISSUE TITLE
the first_out and first_out I generated had problems

### ISSUE THREAD

**Issue by BigbigVegetable** [phillips96/CRA-Inpainting#2]
I trained 300 epochs, but the first_out and first_out I generated had problems, all white and all blue, what did I do wrong, can you give me some suggestions?

![0_output](https://user-images.githubusercontent.com/39050018/113108358-7b095d80-9237-11eb-85f1-5bdd43bc8e79.jpg)

**Comment by phillips96** [comment]
Is this using any custom code or purely the code in the repository? 

Also could I ask what Dataset you're using?

Thanks :)

**Comment by BigbigVegetable** [comment]
I only use the code in the repository, except for the image path, there is no change in other places. The dataset is the CelebA-HQ dataset :)

**Comment by phillips96** [comment]
There's a call in the training loop that occasionally writes out samples of the model as it's going. It should be in a 'Samples' directory. Do they all look blue and white as well?

**Comment by BigbigVegetable** [comment]
The mask I used in training is as follows：
![1](https://user-images.githubusercontent.com/39050018/113255301-40b4c480-92fa-11eb-86f6-e536bc6f3d79.jpg)

Here's a new inference from today's training：
![0_output](https://user-images.githubusercontent.com/39050018/113255320-47433c00-92fa-11eb-9d9a-554d8ff2f95b.jpg)

**Comment by phillips96** [comment]
I'll see if I can replicate this over the weekend, and let you know if I find anything that might help. 
But as it stands, I have no clue how/why that is happening. 

One thing I would change is the `__steps_per_epoch__` flag in the `train.py` script. That defaults to 100, which is extremely low, so I would raise that value anywhere from 1000-20000, based on your GPU setup. I defaulted it low just for testing purposes.

That's unlikely to fix this, but it is worth noting. I'll change the default now.

**Comment by BigbigVegetable** [comment]
OK,I'll try again,too.
Thanks :)

**Comment by LucasKirsten** [comment]
Hi. Any updates on this issue? I am having similar problems in my training.
